packages/remix-server-runtime/data.ts

```diff
@@ -18,7 +20,7 @@ export async function callRouteAction({
   match,
   request,
 }: {
-  loadContext: unknown;
+  loadContext?: AppLoadContext;
```

Comment:
I did not take too much time in learning what the implementation details were of changing this (and the following) types like this. Relying on reviewers to correct me if I'm wrong here.